docs/manual/manual_redundant_throttle.rst

```diff
@@ -0,0 +1,113 @@
+
+Vertiq's redundant throttle support is a safety feature that allows your module to accept multiple throttle sources' commands simultaneously, and to switch between them should 
+the primary source fail. For example, suppose you would like to control your module using the :ref:`DroneCAN protocol <dronecan_protocol>`. To do so, you connect your module's 
+CAN connections with your flight controller's, and connect no other throttle sources. 
+
+.. image:: ../_static/manual_images/redundant_throttle/basic_can_connection.png
+    :width: 50%
+
+Now, in flight, your CAN connection comes loose, and the module and flight controller lose DroneCAN communication. In this case, your vehicle will lose control as your module no longer 
+reacts to new throttle commands, and the module will eventually :ref:`manual_timeout`.
+
+Now, suppose that you connect DroneCAN as before, but now, also connect your module to a DSHOT output on your flight controller.
+
+.. image:: ../_static/manual_images/redundant_throttle/dshot_and_dronecan_connection.png
+    :width: 50%
+
+Under normal operating conditions, the module will apply only throttle commands received via DroneCAN (as determined by your redundant throttle configuration discussed :ref:`below <redundant_throttle_config>`). 
+If we once again sever the DroneCAN connection, however, the module will seamlessly transition to accepting DSHOT commands. By connecting multiple throttle sources, and 
+leveraging Vertiq's redundant throttle feature, you robustify your vehicle's ability to fly safely even after communication failures.
+
+
+.. _redundant_throttle_config:
+
+***************************************
+Redundant Throttle Configurations
+***************************************
+
+There are two types of configurations available with the redundant throttle. The first defines the amount of time that the module waits before switching to another throttle 
+source if the active source goes offline. The second set of parameters defines the priorities of each supported throttle source. You can find the specific parameters 
+at :ref:`throttle_source_manager`.
+
+Vertiq's supported throttle sources are :ref:`DroneCAN <dronecan_protocol>`, :ref:`IQUART <uart_messaging>`, and :ref:`hobby protocols <hobby_protocol>`. Please note that only priority values corresponding to protocols supported by 
+your module have any effect. See your module's family page to see what throttle sources are supported.
+
+Priorities can be valued from 0 to 3. Setting a priority of 0 indicates that the module will ignore all throttle messages received from the configured source. 
+Priority values [1, 3] define the priority of each protocol against the others where 3 defines the highest priority.
+
+.. note:: 
+    Due to Vertiq modules' hardware, it is not possible to use both IQUART and hobby protocols simultaneously. As such, it is only possible to use DroneCAN with one of IQUART 
+    or hobby as redundant throttle sources.

```

Comment:
I may just say "it is not currently possible", since one day maybe we'll fix this.